### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by Panos277** [NVIDIA/NeMo#1492]
Is it possible in a TextClassificationModel to set parameter groups for finetuning, meaning that i want weight decay only for 'bias', 'Layernorm.weight' etc?

**Comment by okuchaiev** [comment]
this isn't something supported out-of-the box. But you can probably write your own PTL callback for that.
